packages/bundle/src/createCognitiveServicesBingSpeechPonyfillFactory.js

```diff
@@ -0,0 +1,28 @@
+import createPonyfill from 'web-speech-cognitive-services/lib/BingSpeech';
```

Comment:
Ponyfill factory is a factory function to create ponyfills. Factory function = function that when called, will create objects. And is an object pattern used to reconfigure set of objects in bulk.

We need a factory function because:

- DirectLineJS will give us different configuration at runtime (per conversation)
- We don't want to bring additional hidden-and-static properties to the W3C spec

So this pattern will create a new set of ponyfills for every configuration DLJS give to us. DLJS don't send us new configuration too often (once per conversation).